Vagrantfile

```diff
@@ -231,5 +249,12 @@ Vagrant.configure("2") do |config|
 		end
 	end
 
+	config.vm.provider :parallels do |_v, override|
```

Comment:
Do we definitely need this section? I would've thought that https://github.com/Chassis/Chassis/blob/main/Vagrantfile#L225-L231 would apply to Parallels as well. It worked on both VB and VMWare the last time I used VMWare. I thought it was a catch all for all providers.

Reply:
Without this these synced folders have dmode/fmode parameters causing parallels to balk and provisioning ends. `fmode` and `dmode` parameters are only usable on VirtualBox and should never be used on other providers

Reply:
Gotcha. I missed that part. Taa!